src/i965_drv_video.c

```diff
@@ -1378,6 +1431,10 @@ i965_suface_external_memory(VADriverContextP ctx,
         obj_surface->cb_cr_width = obj_surface->orig_width / 2;
         obj_surface->cb_cr_height = obj_surface->orig_height / 2;
         obj_surface->cb_cr_pitch = memory_attibute->pitches[1];
+        if (tiling)
```

Comment:
@xhaihao , the above check is added for the NV12/P010.  It is reasonable.
The code in line 1377 is to check the tiling attribute for I420/IYUV surface earlier.